### PR TITLE
Remove the examples page from the docs in favour of one on daml.com

### PR DESCRIPTION
**Pull request by bame-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/configs/pdf/index.rst [hunk 1]
```diff
@@ -99,6 +106,8 @@ Early Access Features
    daml-integration-kit/index
    triggers/index
    tools/visual
+   tools/trigger-service
```
Comment (cocreature) [inline docs/configs/pdf/index.rst:109]: We really need to unify the two indices.

**Review by cocreature — APPROVED** [review]
Thanks!

File: docs/configs/pdf/index.rst (hunk 1 above)
Comment (bame-da) [reply docs/configs/pdf/index.rst:109]: I would if I knew how...

Comment (cocreature) [reply docs/configs/pdf/index.rst:109]: I’ll take a look

**Pull request merged**